tests/wp-unit/include/Core/Metas/Modules/DeleteUserMetaModuleTest.php

```diff
@@ -153,4 +158,224 @@ public function test_delete_usermeta_in_batches() {
 		$meta_deleted = $this->module->delete( $delete_options );
 		$this->assertEquals( 10, $meta_deleted );
 	}
+
+	/**
+	 * Test deletion of user metas from more than one users in a single role.
+	 */
+	public function test_deleting_multiple_users_with_meta_fields_from_single_user_role() {
```

Comment:
Change the test method format to `test_that_` format.

I know that other methods have old format, but do it for all the new methods that you have created.